src/App.js

```diff
@@ -1,40 +1,44 @@
-import './App.css';
-import Header from './Header';
-import React, {useState} from 'react';
-import ToDoList from "./ToDoList";
-import ToDoForm from './ToDoForm';
+import "./App.css";
+import Header from "./components/Header";
+import React, { useState } from "react";
+import ToDoList from "./components/ToDoList";
+import ToDoForm from "./components/ToDoForm";
+import ChooseList from "./components/ChooseList";
 
 function App() {
   const [toDoList, setToDoList] = useState([]);
+  const [toggleStatus, setToggleStatus] = useState("Active");
 
   const handleToggle = (id) => {
-    let mapped = toDoList.map(task => {
-      return task.id === id ? { ...task, complete: !task.complete } : { ...task};
+    let mapped = toDoList.map((task) => {
+      return task.id === id
+        ? { ...task, complete: !task.complete }
+        : { ...task };
     });
     setToDoList(mapped);
-  }
-
-  const handleFilter = () => {
-    let filtered = toDoList.filter(task => {
-      return !task.complete;
-    });
-    setToDoList(filtered);
-  }
+  };
 
   const addTask = (userInput) => {
     let copy = [...toDoList];
```

Comment:
unnecessary variable creation, please fix this

Reply:
addTask function is used further in other components.